src/content/ocaml/editor-note.tex

```diff
@@ -0,0 +1,29 @@
+% !TEX root = ctfp-print.tex
+
+\lettrine[lhang=0.17]{T}{his is the OCaml edition} of \emph{Category Theory for Programmers}.
+It's been a tremendous success, making Bartosz Milewski's blog post series available as a nicely 
+typeset \acronym{PDF}, as well as a hardcover book. There have been numerous contributions made
+to improve the book, by fixing typos and errors, as well as translating the code snippets into
+other programming languages.
+
+I am thrilled to present this edition of the book, containing the original Haskell code, followed by
+its Ocaml counterpart. The Ocaml code snippets were generously provided by 
+\urlref{https://github.com/typelevel/CT_from_Programmers.scala}{Typelevel} contributors, slightly 
+modified to suit the format of this book.
+
+To support code snippets in multiple languages, I am using a \LaTeX{} macro to load the code snippets
+from external files. This allows easily extending the book with other languages, while leaving the 
+original text intact. Which is why you should mentally append the words ``and Ocaml'' whenever you see
+``in Haskell'' in the text.
```

Comment:
The name `OCaml` should be properly capitalized everywhere.

Reply:
Done